File: baseble/src/main/java/com/fcbox/ble/baseble/callback/scan/IScanFilter.java
package com.fcbox.ble.baseble.callback.scan;


import com.fcbox.ble.baseble.model.BluetoothLeDevice;


/**
 * 扫描过滤接口，根据需要实现过滤规则
 */
public interface IScanFilter {
    BluetoothLeDevice onFilter(BluetoothLeDevice bluetoothLeDevice);
}

File: baseble/src/main/java/com/xzy/ble/baseble/callback/scan/IScanFilter.kt
package com.xzy.ble.baseble.callback.scan

import com.xzy.ble.baseble.model.BluetoothLeDevice


/**
 * 扫描过滤接口，根据需要实现过滤规则
 */
interface IScanFilter {
    fun onFilter(bluetoothLeDevice: BluetoothLeDevice): BluetoothLeDevice?
}
